coffee-dto/coffee-dto-base/pom.xml

```diff
@@ -22,6 +22,10 @@
 			<artifactId>jakarta.xml.bind-api</artifactId>
 			<version>${version.jakarta.xml.bind.jaxb-api}</version>
 		</dependency>
+		<dependency>
+			<groupId>jakarta.platform</groupId>
+			<artifactId>jakarta.jakartaee-api</artifactId>
```

Comment:
valoszinu a javax.ws.rs.core csomagok miatt kototted be. Ha igen akkor szandekosan a coffee-dto szinten nem huztuk be a jaxrs csomagot. Annak az elso helye a coffee-rest -ben lenne. A Status enumnak is ott lenne a helye.

A DTO csomagokat igyekszunk a leheto lekisebb fuggosegben tartani.

Reply:
Igen amiatt. Értem. Áttettem a restbe.